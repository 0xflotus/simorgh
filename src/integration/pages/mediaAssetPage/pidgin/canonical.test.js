/**
 * @service pidgin
 * @pathname /pidgin/23248703
 */

import runCrossPlatformTests from '../crossPlatformTests';
import runCanonicalTests from '../canonicalTests';

describe(platform.toUpperCase(), () => {
  describe(pageType, () => {
    runCrossPlatformTests();
    runCanonicalTests();
  });
});
