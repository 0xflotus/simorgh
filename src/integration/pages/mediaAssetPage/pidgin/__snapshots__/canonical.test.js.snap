// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CANONICAL Media Asset Page A11y I can see the skip to content link 1`] = `"Waka go wetin de inside"`;

exports[`CANONICAL Media Asset Page Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/pidgin"`;

exports[`CANONICAL Media Asset Page Footer I can see the footer copyright and external linking text 1`] = `"Â© 2020 BBC. De external site no concern BBC. De way wey we de take go external link."`;

exports[`CANONICAL Media Asset Page Header I can see the branding 1`] = `"BBC News, Pidgin"`;

exports[`CANONICAL Media Asset Page I can see the bulleted list item 1`] = `"<Media Asset Page><Media Asset Page><Media Asset Page>this is the link textthis is the link textthis is the link text"`;

exports[`CANONICAL Media Asset Page I can see the headline 1`] = `"Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < >"`;

exports[`CANONICAL Media Asset Page I can see the related content 1`] = `"Police Arrest 3 ontop Anambra Church AttackGovernor Willie Obiano of Anambra State bin don promise say government go handle the matter sharp, sharp8th August 2017Lagos 'Gay' men Appear for Court, say Dem no dey GuiltyThe men wey dem arrest almost one week ago don say dem no dey guilty, as court order dem to go for sexual rehabilitation.8th August 2017Image gallery, Kenya Elections Pictures: 8 August 2017This na some of the picture wey our eye see as Kenya people go vote for election today8th August 20170:15Audio, BBC News Pidgin One Minute News, 0,15All di local and world tori wey you suppose know in 60 seconds!17th August 2017"`;

exports[`CANONICAL Media Asset Page I can see the timestamp 1`] = `"13 September 2019"`;

exports[`CANONICAL Media Asset Page Media player A11y I can read the media player title 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/cps/pidgin/23248703/p01kx42v/pcm"`;

exports[`CANONICAL Media Asset Page SEO Canonical link 1`] = `"http://localhost:7080/pidgin/23248703"`;

exports[`CANONICAL Media Asset Page SEO Dir attribute 1`] = `"ltr"`;

exports[`CANONICAL Media Asset Page SEO FB admins 1`] = `"100004154058350"`;

exports[`CANONICAL Media Asset Page SEO FB app ID 1`] = `"1609039196070050"`;

exports[`CANONICAL Media Asset Page SEO Heading level 1 1`] = `"Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < >"`;

exports[`CANONICAL Media Asset Page SEO Lang attribute 1`] = `"pcm"`;

exports[`CANONICAL Media Asset Page SEO OG description 1`] = `"This week, CPS media assets were visible in Simorgh for the first time. The pages were built out in a matter of weeks using existing Psammead Components."`;

exports[`CANONICAL Media Asset Page SEO OG image 1`] = `"http://ichef.test.bbci.co.uk/news/1024/branded_pidgin/6FC4/test/_63721682_p01kx435.jpg"`;

exports[`CANONICAL Media Asset Page SEO OG image alt 1`] = `"connectionAltText"`;

exports[`CANONICAL Media Asset Page SEO OG locale 1`] = `"pcm"`;

exports[`CANONICAL Media Asset Page SEO OG site name 1`] = `"BBC News Pidgin"`;

exports[`CANONICAL Media Asset Page SEO OG title 1`] = `"Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < > - BBC News Pidgin"`;

exports[`CANONICAL Media Asset Page SEO OG type 1`] = `"article"`;

exports[`CANONICAL Media Asset Page SEO OG url 1`] = `"http://localhost:7080/pidgin/23248703"`;

exports[`CANONICAL Media Asset Page SEO Page title 1`] = `"Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < > - BBC News Pidgin"`;

exports[`CANONICAL Media Asset Page SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`CANONICAL Media Asset Page SEO Twitter card 1`] = `"summary_large_image"`;

exports[`CANONICAL Media Asset Page SEO Twitter creator 1`] = `"@BBCNews"`;

exports[`CANONICAL Media Asset Page SEO Twitter description 1`] = `"This week, CPS media assets were visible in Simorgh for the first time. The pages were built out in a matter of weeks using existing Psammead Components."`;

exports[`CANONICAL Media Asset Page SEO Twitter image alt 1`] = `"connectionAltText"`;

exports[`CANONICAL Media Asset Page SEO Twitter image src 1`] = `"http://ichef.test.bbci.co.uk/news/1024/branded_pidgin/6FC4/test/_63721682_p01kx435.jpg"`;

exports[`CANONICAL Media Asset Page SEO Twitter site 1`] = `"@BBCNews"`;

exports[`CANONICAL Media Asset Page SEO Twitter title 1`] = `"Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < > - BBC News Pidgin"`;
